### DISCUSSION TITLE
Process finished with exit code 137 (interrupted by signal 9: SIGKILL)

### DISCUSSION THREAD

**Discussion by taogeanton2** [winedarksea/AutoTS#80] [Q&A]
Process finished with exit code 137 (interrupted by signal 9: SIGKILL)

**Comment by winedarksea** [comment]
SIGKILL is where the Python process is terminated by your system. 
Reasons I have seen this:
1. Low resources (not enough RAM, usually) - monitor and see how much the program is using. You might also want to try explicitly setting n_jobs to a low number, as CPU over-subscription could be an issue.
2. Timeouts - for example, when scheduled by Airflow or another system, if it runs past a timeout specified by orchestration
3. User or system terminates the Python process (accidentally or deliberately) from another process

Do you have any more details to help? @taogeanton2

**Reply by taogeanton2** [reply]
Thank you for your suggestion! Setting n_jobs to a low numbe will not report an error! Thanks!
